pkg/cri/server/service_linux.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
Not sure if this just ends up getting ignored, either way it shouldn't necessary. If was being used then it would cause a conflict with `service_other.go`.